lib/core/src/tab/tab.tsx

```diff
@@ -28,14 +28,36 @@ interface TabStyle {
 }
 
 interface Props {
+	/**
+	 * Title and Body of each tab.
+	 */
```

Comment:
Yeah this is good for now. Later we should mention something like "See the detail in Tab tab. It's similar to the Table docs, where "columns" refer to a "TableColumn" interface which is defined in a separated table:

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/5953369/119694909-e195ab00-be77-11eb-9f81-6225f6f96bee.png">